x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/GroupingAggregatorFunctionTestCase.java

```diff
@@ -303,7 +303,7 @@ public final void testMulitvaluedNullGroupsAndValues() {
         assertSimpleOutput(origInput, results);
     }
 
-    public final void testMulitvaluedNullGroup() {
+    public void testMulitvaluedNullGroup() {
```

Comment:
@nik9000 fyi only, I wanted to mute a particular subtest, thats difficult to do with the test methods being final so I changed this temporarily. Hope you don't mind and can change it back, let me know if there are other better ways to do this.